packages/core/src/types/redocly-yaml.ts

```diff
@@ -141,40 +142,6 @@ const ConfigStyleguide: NodeType = {
   },
 };
 
-const RootConfigStyleguide: NodeType = {
```

Comment:
Can we get this and `ConfigRoot` back in **redocly-yaml.ts** and move [this line](https://github.com/Redocly/redocly-cli/pull/1133/files?diff=split&w=1#diff-2528117ea3cbe25dfbfd0583c9b11ba7baa7ef347184dfba32a73ec486aa1a5eR186) to the **config-root.ts** file instead?